spec/goss/goss.yaml

```diff
@@ -1,25 +1,31 @@
 file:
+  /backup-consumer.sh:
+    exists: true
+    owner: root
+    group: root
+    filetype: file
   /create-sa.sh:
     exists: true
-    mode: "0755"
+    owner: root
+    group: root
+    filetype: file
+  /pre-upgrade.sh:
```

Comment:
nice catch to add these to the spec. What about `/upgrade-notification.sh`?

- [ ] done

Reply:
Right, I added that one later and forgot about it. We hve it now since https://github.com/cloudogu/postgresql/pull/9/commits/682c2637a552e0d229043d7d43e15b16567bfec3